Marlin/Marlin_main.cpp

```diff
@@ -4055,11 +4066,17 @@ inline void gcode_M206() {
    *    L = diagonal rod
    *    R = delta radius
    *    S = segments per second
+   *    A = Alpha (Tower 1) diagonal rod trim
+   *    B = Beta (Tower 2) diagonal rod trim
+   *    G = Gamma (Tower 3) diagonal rod trim
    */
   inline void gcode_M665() {
     if (code_seen('L')) delta_diagonal_rod = code_value();
     if (code_seen('R')) delta_radius = code_value();
     if (code_seen('S')) delta_segments_per_second = code_value();
+    if (code_seen('A')) delta_diagonal_rod_trim_tower_1 = code_value();
+    if (code_seen('B')) delta_diagonal_rod_trim_tower_2 = code_value();
+    if (code_seen('G')) delta_diagonal_rod_trim_tower_3 = code_value();
```

Comment:
`G` may be controversial as a parameter because the letter G is reserved for the G-code commands that start with `G`. While most parsers can probably handle the parameter just fine, for maximum compliance with everyone's G-code standards we should consider another letter. The letter `C` is okay. `ABC` would also correspond with the lettering we'll probably be using in the future when referring directly to the delta tower (scara / corexy / corexz) steppers in commands.